projects/packages/masterbar/src/admin-menu/class-base-admin-menu.php

```diff
@@ -497,7 +497,7 @@ public function hide_parent_of_hidden_submenus() {
 			$has_submenus = isset( $submenu[ $menu_item[2] ] );
 
 			// Skip if the menu doesn't have submenus.
-			if ( ! $has_submenus || ! is_array( $submenu[ $menu_item[2] ] ) ) {
+			if ( ! $has_submenus || ! is_array( $submenu[ $menu_item[2] ] ) || empty( $submenu[ $menu_item[2] ] ) ) {
```

Comment:
(GH will probably mangle this suggestion, but)
```suggestion
			$has_submenus = ! empty( $submenu[ $menu_item[2] ] );

			// Skip if the menu doesn't have submenus.
			if ( ! $has_submenus || ! is_array( $submenu[ $menu_item[2] ] ) ) {
```
Or consolidate it even further
```suggestion
			// Skip if the menu doesn't have submenus.
			if ( empty( $submenu[ $menu_item[2] ] ) || ! is_array( $submenu[ $menu_item[2] ] ) ) {
```

Reply:
Good point, Brad, thanks! Done with https://github.com/Automattic/jetpack/pull/38363/commits/531902443f728f7795a02e48cfc7746de9651a74